### PR TITLE
Reset /etc/motd to include version, a pointer to source image and GitHub

### PR DESCRIPTION
**Pull request by zmerlynn**
Companion to #21031

### CODE REVIEW

**Comment by zmerlynn** [comment]
Sample output for a normal release:
```
Welcome to Kubernetes v99.99.99!

You can find documentation for this release at:
  http://kubernetes.io/v99.99/

You can download the build image for this release at:
  https://storage.googleapis.com/kubernetes-release/release/v99.99.99/kubernetes-src.tar.gz

It is based on the Kubernetes source at:
  https://github.com/kubernetes/kubernetes/tree/v99.99.99
```

Sample output for a development build (or a CI build):
```
You can find documentation for this release at:
  http://kubernetes.io/v1.2/

You can download the build image for this release at:
  https://storage.googleapis.com/kubernetes-release/release/v1.2.0-alpha.7.622+b857f61009deb4-dirty/kubernetes-src.tar.gz

It is based on the Kubernetes source at:
  https://github.com/kubernetes/kubernetes/tree/b857f61009deb4-dirty

Note: This looks like a development version, which might not be present on GitHub.
If it isn't, the closest tag is at:
  https://github.com/kubernetes/kubernetes/tree/v1.2.0-alpha.7
```

**Comment by zmerlynn** [comment]
Also note: this relies on #21031 for the `https://storage.googleapis.com/kubernetes-release/release/v99.99.99/kubernetes-src.tar.gz` link to work, but it can be merged independently, as long as they both make it in.

File: cluster/gce/configure-vm.sh [hunk 1]
```diff
@@ -74,8 +74,37 @@ function set-broken-motd() {
   echo -e '\nBroken (or in progress) Kubernetes node setup! Suggested first step:\n  tail /var/log/startupscript.log\n' > /etc/motd
 }
 
-function set-good-motd() {
-  echo -e '\n=== Kubernetes node setup complete ===\n' > /etc/motd
+function reset-motd() {
+  # kubelet is installed both on the master and nodes, and the version is easy to parse (unlike kubectl)
+  local -r version="$(/usr/local/bin/kubelet --version | cut -f2 -d\ )"
```
Comment (roberthbailey) [inline cluster/gce/configure-vm.sh]: Older versions of the kubelet want `--version=true` here. Do we care if this only works for 1.2+?

Comment (roberthbailey) [inline cluster/gce/configure-vm.sh]: Does `(... | cut -f2 -d\ )` cut along spaces? If so, I think `(... | cut -f2 -d " ")` is easier to read.

Comment (zmerlynn) [inline cluster/gce/configure-vm.sh]: Fixed both. (And yeah, `-d\ ` is just a short way to get the space.)

**Comment by roberthbailey** [comment]
LGTM.

**Pull request merged**